src/js/controllers/tab-scan.js

```diff
@@ -57,68 +68,133 @@ angular.module('copayApp.controllers').controller('tabScanController', function(
 
 
   $scope.$on("$ionicView.beforeEnter", function(event, data) {
-    console.log('tab-scan $ionicView.beforeEnter');
-    console.log('data', data.stateParams.returnRoute);
-    _handleCapabilities();
-    $timeout(() => {
-        _refreshScanView()
-        if (isSafari() !== "SAFARI") {
-          $scope.cameraIssue = true
-        }
-      },
-    5000);
-    $scope.returnRoute = data.stateParams.returnRoute || false;
+    if (platformInfo.cameraType === 'native') {
+      _handleCapabilities();
```

Comment:
why do we check if the camera is native twice (before we call `_handleCapabilities` and then again when we are inside that func). Do we call it from other places so we need to double check just to be sure?